### PR TITLE
Reader: ensure button block matches main site styles

### PR DESCRIPTION
**Pull request by tophertoy**
#### Changes proposed in this Pull Request

This change handles styling the button block in reader to match the website It is only part 1 of 8 of this issue:

https://github.com/Automattic/wp-calypso/issues/43595

#### Testing instructions

Before:

Screenshot 2020-07-01 13 50 54

Taken from: http://calypso.localhost:3000/read/blogs/159889361/posts/1363

Firefox After:

Screenshot 2020-07-01 15 17 40

### CODE REVIEW

**Comment by tophertoy** [comment]
@leandroalonso Here's what I have so far.



Do you have thoughts on what styling you want brought over?

I believe reader is opinionated on colours, so I haven't brought those over.

**Comment by leandroalonso** [comment]
Hi @tophertoy! Thanks for your contribution!

@bluefuton would you mind reviewing this one?

**Comment by bluefuton** [comment]
Thanks for the contribution @tophertoy! Can we match the styles applied to the default Calypso button?

https://wordpress.com/devdocs/packages/components/src/button/README.md?term=button

I think the 'non-primary' version would make the most sense. It looks like this:

Screen Shot 2020-07-01 at 11 29 27

**Comment by tophertoy** [comment]
@bluefuton that link appears not to be working for me. Also https://wordpress.com/devdocs/  is 404ing. 

Do you want me to work from the attached "Bananas" image?

**Comment by bluefuton** [comment]
Sorry @tophertoy - try on wpcalypso:

https://wpcalypso.wordpress.com/devdocs/packages/components/src/button/README.md?term=button

**Comment by tophertoy** [comment]
@bluefuton ok I've made some changes. Attached screenshots from firefox and chrome at the top. 

Let me know what you think or if there is a defined browser list I should check.

File: client/blocks/reader-full-post/_content.scss [hunk 1]
```diff
@@ -241,6 +241,23 @@
 		margin: auto;
 	}
 
+	.wp-block-button {
+		border-style: solid;
```
Comment (bluefuton) [inline client/blocks/reader-full-post/_content.scss:245]: Let's add `font-family: $sans` to match the Button component.

File: client/blocks/reader-full-post/_content.scss [hunk 2]
```diff
@@ -241,6 +241,23 @@
...
+		overflow: hidden;
+		vertical-align: top;
+		font-size: 14px;
+		font-weight: 600;
+		line-height: 22px;
+		border-radius: 2px;
+		padding: 8px 14px;
+		border-color: var( --color-neutral-10 );
+	}
+
```
Comment (bluefuton) [inline client/blocks/reader-full-post/_content.scss:260]: Let's add some styles to remove the link colour from the link - how about:

```
.wp-block-button__link, .wp-block-button__link:hover {
  color: inherit;
}
```

**Comment by tophertoy** [comment]
@bluefuton I've made those changes.

**Comment by bluefuton** [comment]
Just one last tweak needed I think - let's add a touch of right and bottom margin to prevent multiple buttons running into each other:

Screen Shot 2020-07-01 at 15 44 53

**Comment by tophertoy** [comment]
Gap is showing as 8px between buttons on right and below now:



Screenshot 2020-07-02 13 43 12

Thoughts?

Screenshot 2020-07-02 13 43 01

File: client/blocks/reader-full-post/_content.scss [hunk 3]
```diff
@@ -241,6 +241,35 @@
 		margin: auto;
 	}
 
+	.wp-block-buttons {
+		margin: 0px;
```
Comment (bluefuton) [inline client/blocks/reader-full-post/_content.scss:245]:
```suggestion
		margin: 0;
```

Just a nit - zero doesn't need units :)

**Pull request merged**